packages/vega-parser/src/parsers/guides/guide-util.js

```diff
@@ -86,3 +87,80 @@ export function extendOffset(value, offset) {
     : !isObject(value) ? { value, offset }
     : { ...value, offset: extendOffset(value.offset, offset) };
 }
+
+const DISCRETE_SCALES = new Set(['ordinal', 'band', 'point']);
```

Comment:
What about in BinOrdinal? Can we use `isDiscrete` from vega-scales instead?

Reply:
Ahh, thanks. I didn't think of `isDiscrete`.